server/command/service.go

```diff
@@ -108,11 +100,34 @@ func (s *service) allSubCommands(conf config.Config) map[string]commandHandler {
 		}
 	}
 
+	all["uninstall"] = s.uninstallCommand(conf)
 	all["install"] = s.installCommand(conf)
 
 	return all
 }
 
+func (s *service) uninstallCommand(conf config.Config) commandHandler {
+	h := commandHandler{
+		autoComplete: &model.AutocompleteData{
+			Trigger:  "uninstall",
+			HelpText: "Uninstall an App.",
+			RoleID:   model.SYSTEM_ADMIN_ROLE_ID,
+		},
+		subCommands: map[string]commandHandler{},
+	}
+
+	installed := s.proxy.GetInstalledApps()
+	for _, app := range installed {
+		appAC := model.NewAutocompleteData(string(app.Manifest.AppID), "", "Uninstall "+app.Manifest.DisplayName)
```

Comment:
If I am not mistaken, this will create on plugin activate the autocomplete data with the currently installed apps, but will not update. Why not use a dynamic select argument?